index.js

```diff
@@ -25,13 +25,31 @@ var UNICODE_CHARS = {
     '\u2029': '\\u2029'
 };
 
+// We can‘t just instanceof Date since dates are already converted to strings
+// because of native Date.prototype.JSON (which use toISOString)
+var DATE_LENGTH = new Date().toISOString().length
```

Comment:
I disagree. I think setting it to `24` with a comment that it'll always be `24` is a better approach. But this is likely moot given my two comments above: https://github.com/yahoo/serialize-javascript/pull/16#discussion_r62214902